app/controllers/api/migrations.php

```diff
@@ -309,7 +310,7 @@
     ->label('sdk.response.type', Response::CONTENT_TYPE_JSON)
     ->label('sdk.response.model', Response::MODEL_MIGRATION)
     ->param('resources', [], new ArrayList(new WhiteList(NHost::getSupportedResources())), 'List of resources to migrate')
-    ->param('subdomain', '', new URL(), 'Source\'s Subdomain')
+    ->param('subdomain', '', new Text(512), 'Source\'s Subdomain')
```

Comment:
Can we use a host or domain validator instead? Why just a simple text, what do we expect here?